js/src/views/Dapps/Summary/summary.js

```diff
@@ -38,7 +38,7 @@ export default class Summary extends Component {
       return null;
     }
 
-    const url = `/app/${app.local ? 'local' : 'global'}/${app.url}`;
+    const url = `/app/${app.local ? 'local' : 'global'}/${app.id}`;
```

Comment:
Does this actually work now? For built-ins the id is the sha3, so in the dapp we may try to load sha3.html?

We need the hash id, otherwise we cannot reliably register images.